packages/docusaurus-migrate/src/index.ts

```diff
@@ -476,7 +477,7 @@ function handleVersioning(
     const loadedVersions: Array<string> = JSON.parse(
       String(fs.readFileSync(path.join(siteDir, 'versions.json'))),
     );
-    fs.copyFile(
+    fs.copyFileSync(
```

Comment:
good catch :D

Reply:
I should thank to static code analysis build-in IDE 😉 There was a warning about unhandled `Promise`.